src/churn/mocks.js

```diff
@@ -1,9 +1,14 @@
 // Create a mock instance of Freedom.
 // We do this in a non-TypeScript file because the ambient module declaration
 // prevents us creating any variable called freedom in TypeScript-land.
-var freedom = jasmine.createSpyObj('freedom', ['core.log']);
-freedom.churn = jasmine.createSpy().and.returnValue(
-    jasmine.createSpyObj('churn', ['providePromises']));
-freedom['core.console'] = function() {
-  return window.console;
+var freedom = {
```

Comment:
There is, and @iislucas is working on this as part of his big require change. So, we can benefit from that when the time comes to update this repo.